### PR TITLE
Update misc webpack related packages

### PR DESCRIPTION
**Pull request by CraigMacomber**
As part of the effort to update to webpack5 (#9568) this updates a few webpack related packages, hopefully reducing the number of issues when updating webpack itself.

The most significant change is migrating from duplicate-package-checker-webpack-plugin to its fork @cerner/duplicate-package-checker-webpack-plugin because the original has not merged support for webpack5, see https://github.com/darrenscerri/duplicate-package-checker-webpack-plugin/pull/46

### CODE REVIEW

File: examples/apps/collaborative-textarea/webpack.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 const fluidRoute = require("@fluid-tools/webpack-fluid-loader");
 const path = require("path");
-const merge = require("webpack-merge");
+const { merge } = require("webpack-merge");
```
Comment (heliocliu) [inline examples/apps/collaborative-textarea/webpack.config.js:8]: It looks like this is only used to merge the dev/prod setting
@tylerbutler Do you know if our examples are ever used in a way that broadly requires that dev/prod split?  If not it seems like that may be worth cleaning up for the examples.

**Review by heliocliu — APPROVED** [review]

**Comment by CraigMacomber** [comment]
> Found some broken links!

I checked, and these links now work, so I think this was a temporary external issue, and not due to web packing the docs differently.

**Comment by CraigMacomber** [comment]
debugger seems to be failing to find the "fs" module, which should be provided by the "@types/node" that is has.
For some reason (which I have no idea about), it did not need  `"types": [ "node" ]` in the tsconfig before, but needs it now.

This is now fixed.

**Comment by tylerbutler** [comment]
> > Found some broken links!
> 
> I checked, and these links now work, so I think this was a temporary external issue, and not due to web packing the docs differently.

Yeah, I've seen these failures intermittently before. You can ignore them.

**Pull request merged**